components/list/demo/list-item-sample.js

```diff
@@ -0,0 +1,50 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ListItemCheckboxMixin } from '../list-item-checkbox-mixin.js';
+//import { ListItemDragMixin } from '../list-item-drag-mixin.js';
+
+class ListItemSample extends ListItemCheckboxMixin(LitElement) {
+	static get properties() {
+		return {
+			breakpoints: { type: Array },
+			disabled: {type: Boolean },
+			href: { type: String },
+			key: { type: String, reflect: true },
+			role: { type: String, reflect: true },
+			_breakpoint: { type: Number }
+		};
+	}
+
+	static get styles() {
+		return [ (super.styles ? super.styles : css``), css`
```

Comment:
I am curious if this is necessary?  I know properties of the mixin are automatically mixed with the subclass, but I am not sure about the styles.  In other places we'd placed the styles in a separate file and consumers import them, which isn't ideal from a request perspective.

Reply:
The styles are not mixed automatically - this is something we're doing in other places: https://github.com/BrightspaceUI/core/blob/master/components/hierarchical-view/hierarchical-view.js#L7

It seemed cleaner than putting the styles in a separate file when they are only related to the mixin, but I could be convinced of another way

Reply:
Ok thanks.  Yeah I see menu does this as well, with the hierarchical-view as you mention.  I think there are other places we may similarly optimize to avoid a separate HTTP request.  Something I will look into.  :)